.github/dependabot.yml

```
@@ -1,3 +1,12 @@
 # Basic set up for package managers
 version: 2
-updates:
\ No newline at end of file
+updates:
+    - package-ecosystem: "gomod"
+      directory: "/go/"
+      reviewers:
+        - "AndrewGutierrezCastro"
+        - "maurigamg"
+        - "Marcosmh0199"
+        - "danielcdz"
```

You can use the team instead 👀 
```suggestion
      reviewers:
        - "data-engineering-samples/reviewers"
```

---

Done @maurigamg